bundle/bundle.go

```diff
@@ -32,6 +32,7 @@ import (
 const (
 	RegoExt               = ".rego"
 	WasmFile              = "policy.wasm"
+	PlanFile              = "plan.json"
```

Comment:
Is this plan file something that users create themselves or it's created as part of the compile step ?

Reply:
The planner generates the plan and then the `compile` package writes it into the bundle.